src/ngraph/graph_util.cpp

```diff
@@ -355,6 +355,33 @@ pair<shared_ptr<op::Result>, shared_ptr<op::Parameter>>
     return make_pair(res_node, par_node);
 }
 
+pair<shared_ptr<op::Result>, shared_ptr<op::Parameter>>
```

Comment:
Add a comment to these functions explaining that they are temporary to handle where placement will be changed from enum to size_t. Add this comment to all *_size functions